### DISCUSSION TITLE
How to use on a TCP port instead of PCSC

### DISCUSSION THREAD

**Discussion by Hichemslh** [martinpaljak/GlobalPlatformPro#354] [Q&A]
Hi , 
I was wondering if it’s possible to use GlobalPlatformPro( or Apdu4j ) on a virtual smart card to which we connect using a TCP port on localhost (local machine) instead of a real card on a reader ( so there is no PCSC ) ? 
if it’s possible , can someone help me do it ? 
Thank you

**Comment by martinpaljak** [comment]
Absolutely possible and depends on what kind of emulator you're using or what do you want to achieve. It is all software, so anything is possible, question is if it should be done or how it should be done. You can e-mail me.
